data/blog/advancements-in-bitcoin-and-lightning-wallets.mdx

```diff
@@ -0,0 +1,395 @@
+depending on intermediaries. As the network grows, these tools continue to
+evolve toward stronger privacy, smoother interoperability, and better usability.
+
+Over the past year, a new wave of development has advanced both on-chain and
+Lightning self-custody. Developers have improved the speed and reliability of
+mobile and desktop wallets, added advanced features once reserved for experts,
+and made complex Bitcoin functions more intuitive. This progress shows that ease
+of use and technical depth can advance together without compromising security or
+control.
+
+With support from OpenSats, the six projects featured in this impact report
+represent advances in Bitcoin and Lightning wallet development.
+
+The projects highlighted in this impact report are:
+
+- [Bitcoin Safe](#bitcoin-safe)
+- [Blixt Wallet](#blixt-wallet)
+- [Blitz Wallet](#blitz-wallet)
+- [Clams](#clams)
+- [Cove](#cove)
+- [Satsigner](#satsigner)
+
+The above list is not exhaustive. OpenSats has supported many more wallet
+projects, which we plan to highlight in future impact reports. We are also
+providing funding to several [ecash wallets](/blog/advancements-in-ecash).
+
+These projects demonstrate the ongoing refinement of self-custody across the
+Bitcoin ecosystem. From full-node operators to mobile-first users, developers
+are closing the gap between technical sophistication and everyday reliability.
+The result is a more capable and accessible network built on verifiable code and
+open collaboration.
+
+Let's take a closer look at how each of these projects is making an impact.
+
+---
+
+### Bitcoin Safe
+
+[Bitcoin Safe] is a savings wallet built for secure, private, long-term
+self-custody. It helps users manage their bitcoin holdings through a simple
+interface while maintaining control of funds.
+
+Funding [from OpenSats] enabled ongoing improvements in reliability and
+transparency. The wallet now connects directly to Bitcoin's network for [live
+transaction notifications], removing the need for centralized servers. Users can
+see on-chain confirmations in real time while preserving privacy.
+
+[Support] for hardware wallets like Blockstream Jade Plus expanded compatibility
+and security. The wallet also [warns] about look-alike address scams and tracks
+[unconfirmed transactions] to reduce risk.
+
+Behind the interface, Bitcoin Safe is prepared for full adoption of [`Compact
+Block Filters`] (`BIP157/158`), which let wallets identify their own
+transactions privately without querying external servers. This brings privacy
+closer to that of full nodes while remaining lightweight, requiring less storage
+or processing power. Project developers collaborated with the Bitcoin Dev Kit
+(`BDK`) community to refine handling of unconfirmed transactions, address
+lookahead, and encrypted storage, helping improve BDK-based wallets overall.
+
+>Compact Block Filters offer fast performance while removing centralization
```

Comment:
Fetching block filters is usually from a (centralized) server. Unless you are fetching the block filters from your own node. IDK if this is the case with this wallet.